src/app/common/types.ts

```diff
@@ -0,0 +1,23 @@
+// TODO: lib-ui candidate
+export interface MetaTypeMeta {
+  apiVersion: string;
+  kind: string;
+}
```

Comment:
Moving common TS types to lib-ui is a great idea! Hmm, I wonder if that would work since we compile to JS with Rollup. It exports `.d.ts` files too, maybe that would be sufficient.

Reply:
@mturley CRs are so omnipresent that any generalization could help. Not sure that's the best way but would help if we could go in that direction.